### PR TITLE
[T4A2][T10-B2] Low Yong Cheng

### PR DESCRIPTION
**Pull request by yongchenglow**
Converted Parser to a class variable
Added a sequence number to Person

### CODE REVIEW

**Comment by yongchenglow** [comment]
Ready for Review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
```
Comment (nirandiw) [inline src/seedu/addressbook/data/person/Person.java:17]: Where is the getter for this private variable?
Consider adding the getter to the interface ReadOnlyPerson if you want to give access to that as well.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```
Comment (nirandiw) [inline src/seedu/addressbook/parser/Parser.java:53]: Change the call of this method to suite a static method call in Main.java

**Comment by nirandiw** [comment]
@yongchenglow Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**